### ISSUE TITLE
Are you planning a new fbv release?

### ISSUE THREAD

**Issue by artyom-poptsov** [godspeed1989/fbv#28]
A  proper release marked by a Git signed and annotated tag would be helpful for packagers.

**Comment by ffontaine** [comment]
Indeed, a new release would be great: I could then update fbv package on buildroot.
